wright/src/grammar/parsers/expression/index_expression.rs

```diff
@@ -0,0 +1,58 @@
+use crate::grammar::ast::eq::ASTEq;
+use crate::grammar::ast::{Expression, IndexExpression};
+use crate::grammar::model::{Fragment, HasFragment};
+use crate::grammar::parsers::expression::ToExpression;
+use crate::grammar::parsers::whitespace::token_delimiter;
+use crate::grammar::parsers::with_input;
+use nom::character::complete::char as ch;
+use nom::combinator::map;
+use nom::sequence::{delimited, pair, terminated};
+use nom::IResult;
+
+impl<'s> IndexExpression<'s> {
+    /// Square brace characters. Probably should never change.
+    pub const SQUARE_BRACKETS: (char, char) = ('[', ']');
```

Comment:
Consider `DELIMITERS` as that is their usage

Reply:
Done!